contributor/outline.md

```diff
@@ -67,12 +68,13 @@ Outline for this section inspired by the [Trusted Committer introduction video].
     - Feedback from the host team is not any kind of personal criticism to you.
     Treat and value it as data to help your contribution become all the better.
     - If the host team decides not to accept a contribution, don't take it personally.
+    Work with them to understand their reasoning.
     Remember and respect that the host team has ultimate decision over the project.
     They're the ones that are signing up to maintain it, after all!
     Try to collaborate on an aspect of the contribution that you all agree on,
     and look for another way to fill the rest of your needs.
   
-## Mechanics of Contributing
```

Comment:
Why did we change this?  I think 'Mechanics of Contributing' actually works better as a title...

Reply:
This was feedback from @gruetter on [the previous PR](https://github.com/InnerSourceCommons/InnerSourceLearningPath/pull/59).  Although on second thought I think I agree with you.

Reply:
I would agree with @rrrutledge and @guywmartin. Also, I think that mechanics would describe the happy path / clean case of contributions, while the _Gotchas_ surely exist and to me would describe corner cases and other non-standard things.